Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -104,6 +104,7 @@ open class NavigationMapView: UIView {
     var fractionTraveled: Double = 0.0
     var preFractionTraveled: Double = 0.0
     var vanishingRouteLineUpdateTimer: Timer? = nil
+    var showSpecificLegIndex: Int?
```

Comment:
What about 
```suggestion
    var currentLegIndex: Int?
```
current naming looks more like a flag.

Reply:
Thanks for your suggestion! I'll fix this part.